cmd/keycloakb/keycloak_bridge.go

```diff
@@ -855,7 +880,7 @@ func main() {
 		}
 
 		// new module for KYC service
-		kycComponent := kyc.NewComponent(technicalTokenProvider, registerRealm, keycloakClient, usersDBModule, eventsDBModule, accredsModule, kycLogger)
+		kycComponent := kyc.NewComponent(technicalTokenProvider, registerRealm, keycloakClient, usersDBModule, archiveDBModule, eventsDBModule, accredsModule, kycLogger)
```

Comment:
Users can also be modified by validation API (IDnow), so maybe we should also have archiveDBModule for it no?

Reply:
Done